WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -150,3 +187,31 @@ $venue-map-wrapper-height: 400px;
 .big-column {
   width: 100%;
 }
+
+.competition-select {
+  input,
+  select {
+    font-size: 12px;
+    font-weight: normal;
+  }
+
+  button {
+    padding: 2px 8px;
+    font-size: 14px;
+    width: 100px;
+  }
+}
+
+.same-line {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+.table-striped>tbody>tr:nth-child(odd)>th {
+  background-color: #ccc;
```

Comment:
What's going on here? I'm guessing you don't like our table-striped color? If so, can't this be done by changing a bootstrap variable? More importantly, lets do this in a separate PR, where we can debate just the merits of this design change.